### PR TITLE
[Spectrum] Added a section about DDoS Protection for Spectrum

### PR DESCRIPTION
**Pull request by angelampcosta**

### CODE REVIEW

File: content/spectrum/about/ddos-for-spectrum.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+pcx_content_type: concept
+title: DDoS Protection for Spectrum
+weight: 0
+---
+
+# DDoS Protection for Spectrum application
+
+Spectrum provides DDoS Protection at layers 3-4 of the [OSI model](https://www.cloudflare.com/learning/ddos/glossary/open-systems-interconnection-model-osi/), that is against TCP and UDP based DDoS attacks.
+
+Spectrum works as a layer 4 reverse proxy, therefore a proper TCP connection must be first established before traffic is proxied to the origin. This moves any impact of SYN or SYN-ACK reflection attacks to the Cloudflare edge. Additionally, by using Spectrum in front of your application, your origin IP is concealed — preventing attackers from targeting your origin server directly. It is also recommended that you replace your origin IP address after moving to Cloudflare, and lock it down to only accept traffic from Cloudflare’s IP address range.
```
Comment (oyo-lab) [inline content/spectrum/about/ddos-for-spectrum.md]: Can we link "Cloudflare's IP address range" to https://www.cloudflare.com/ips/

File: content/spectrum/about/ddos-for-spectrum.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+Random or out-of-state TCP packets should not be passed to the origin if a legitimate TCP connection has not yet been established between the client and Cloudflare. Spectrum also [leverages SYN cookie challenges as part of the Linux networking stack](https://blog.cloudflare.com/syn-packet-handling-in-the-wild/) to defend against floods.
+
+Furthermore, if a flood of packets of an unspecified protocol target your application, (for example, your Spectrum application is for TCP traffic, and a UDP flood targets your Spectrum application), the packets will be dropped. Similarly, if packets target a port or port range that you did not specify, they will also be dropped.
+
+L3/4 DDoS attack should be detected and mitigated by the [Network-layer DDoS Managed Ruleset](/ddos-protection/managed-rulesets/network/) that is enabled by default. This ruleset detects and mitigates DDoS attacks by dynamically fingerprinting attacks based on packet header fields.
```
Comment (pedrosousa) [inline content/spectrum/about/ddos-for-spectrum.md]:
```suggestion
L3/4 DDoS attacks should be detected and mitigated by the [Network-layer DDoS Managed Ruleset](/ddos-protection/managed-rulesets/network/) that is enabled by default. This ruleset detects and mitigates DDoS attacks by dynamically fingerprinting attacks based on packet header fields.
```

File: content/spectrum/about/ddos-for-spectrum.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+For protecting HTTP/S applications against L7 DDoS attacks and to benefit from caching and additional features, onboard your application to Cloudflare’s Web Application Firewall/Content Delivery Network service — which works in tandem with Cloudflare Spectrum.
```
Comment (pedrosousa) [inline content/spectrum/about/ddos-for-spectrum.md]:
```suggestion
For protecting HTTP/S applications against L7 DDoS attacks and to benefit from caching and additional features, onboard your application to Cloudflare’s Web Application Firewall/Content Delivery Network service, which works in tandem with Cloudflare Spectrum.
```

File: content/spectrum/about/ddos-for-spectrum.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+Refer to [Cloudflare DDoS Protection](https://developers.cloudflare.com/ddos-protection/), to learn more.
```
Comment (pedrosousa) [inline content/spectrum/about/ddos-for-spectrum.md]:
```suggestion
Refer to [Cloudflare DDoS Protection](/ddos-protection/) to learn more.
```

File: content/spectrum/_index.md [hunk 5]
```diff
@@ -8,7 +8,7 @@ meta:
 
 # Cloudflare Spectrum
 
-Spectrum provides security and acceleration for any [TCP](https://www.cloudflare.com/learning/ddos/glossary/tcp-ip/) or [UDP](https://www.cloudflare.com/learning/ddos/glossary/user-datagram-protocol-udp/) based application. That means you can route MQTT, email, file transfer, version control, games, and more over TCP or UDP through Cloudflare to mask the origin and protect it from [DDoS](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/).
+Spectrum provides security and acceleration for any [TCP](https://www.cloudflare.com/learning/ddos/glossary/tcp-ip/) or [UDP](https://www.cloudflare.com/learning/ddos/glossary/user-datagram-protocol-udp/) based application. That means you can route MQTT, email, file transfer, version control, games, and more over TCP or UDP through Cloudflare to mask the origin and protect it from [DDoS attacks](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/). Refer to [DDos Protection for Spectrum application](/spectrum/about/ddos-for-spectrum/), to learn more about what L3/4 DDoS Protection is included as part of the Spectrum service.
```
Comment (pedrosousa) [inline content/spectrum/_index.md]:
```suggestion
Spectrum provides security and acceleration for any [TCP](https://www.cloudflare.com/learning/ddos/glossary/tcp-ip/) or [UDP](https://www.cloudflare.com/learning/ddos/glossary/user-datagram-protocol-udp/) based application. That means you can route MQTT, email, file transfer, version control, games, and more over TCP or UDP through Cloudflare to mask the origin and protect it from [DDoS attacks](https://www.cloudflare.com/learning/ddos/what-is-a-ddos-attack/). Refer to [DDoS Protection for Spectrum application](/spectrum/about/ddos-for-spectrum/) to learn more about what L3/4 DDoS Protection is included as part of the Spectrum service.
```

File: content/spectrum/about/ddos-for-spectrum.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+pcx_content_type: concept
+title: DDoS Protection for Spectrum
+weight: 0
+---
+
+# DDoS Protection for Spectrum application
```
Comment (pedrosousa) [inline content/spectrum/about/ddos-for-spectrum.md]: I'd remove "application" or make it plural ("applications").

File: content/spectrum/about/ddos-for-spectrum.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Random or out-of-state TCP packets should not be passed to the origin if a legitimate TCP connection has not yet been established between the client and Cloudflare. Spectrum also [leverages SYN cookie challenges as part of the Linux networking stack](https://blog.cloudflare.com/syn-packet-handling-in-the-wild/) to defend against floods.
+
+Furthermore, if a flood of packets of an unspecified protocol target your application, (for example, your Spectrum application is for TCP traffic, and a UDP flood targets your Spectrum application), the packets will be dropped. Similarly, if packets target a port or port range that you did not specify, they will also be dropped.
```
Comment (pedrosousa) [inline content/spectrum/about/ddos-for-spectrum.md]:
```suggestion
Furthermore, if a flood of packets of an unspecified protocol target your application (for example, your Spectrum application is for TCP traffic, and a UDP flood targets your Spectrum application), the packets will be dropped. Similarly, if packets target a port or port range that you did not specify, they will also be dropped.
```

**Review by pedrosousa — APPROVED** [review]

File: content/spectrum/_index.md (hunk 5 above)
Comment (pedrosousa) [reply content/spectrum/_index.md]: Same comment here about removing "application" or changing it to "applications" (in the link text).

**Pull request merged**